components/FileUploader.js

```diff
@@ -114,39 +114,45 @@ class Filename extends Component {
   };
 
   render() {
-    const { iconDescription, status, style, tabIndex, onKeyDown, ...other } = this.props;
-    const tempStyle = Object.assign(style, { marginRight: '-1px' }); // temp style correction for loading component position
-    return (
-      <span>
-        {status === 'uploading'
-          ? <div className="bx--loading" style={tempStyle} {...other}>
-              <svg className="bx--loading__svg" viewBox="-42 -42 84 84">
-                <circle cx="0" cy="0" r="37.5" />
-              </svg>
-            </div>
-          : null}
-        {status === 'edit'
-          ? <Icon
-              className="bx--file-close"
-              name="close--glyph"
-              description={iconDescription}
-              style={style}
-              tabIndex={tabIndex}
-              onKeyDown={onKeyDown}
-              {...other}
-            />
-          : null}
-        {status === 'complete'
-          ? <Icon
-              className="bx--file-complete"
-              name="checkmark--glyph"
-              description={iconDescription}
-              style={style}
-              {...other}
-            />
-          : null}
-      </span>
-    );
+    const { iconDescription, status, style, ...other } = this.props;
+
+    if (status === 'uploading') {
+      return (
+        <div
+          className="bx--loading"
+          style={Object.assign(style, { width: '1rem', height: '1rem' })}
+          {...other}
+        >
+          <svg className="bx--loading__svg" viewBox="-42 -42 84 84">
+            <circle cx="0" cy="0" r="37.5" />
+          </svg>
+        </div>
+      );
+    } else if (status === 'edit') {
+      return (
+        <Icon
+          iconDescription={iconDescription}
```

Comment:
Hey, @asudoh, checking this now and I'm not seeing this error. It may have been there before but it's resolved now.